sphinx/ext/autosummary/generate.py

```diff
@@ -197,6 +197,21 @@ def get_members(obj: Any, types: Set[str], include_public: List[str] = [],
             get_members(obj, {'class'}, imported=imported_members)
         ns['exceptions'], ns['all_exceptions'] = \
             get_members(obj, {'exception'}, imported=imported_members)
+
+        # Find module attributes with docstrings
+        attrs, public = [], []
+        from sphinx.pycode import ModuleAnalyzer, PycodeError
+        try:
+            analyzer = ModuleAnalyzer.for_module(name)
+            attr_docs = analyzer.find_attr_docs()
+            for _, attr_name in attr_docs:
+                if attr_name in ns['members']:
```

Comment:
This is not a good way to filter the documented attributes. The key of `attr_docs` is pairs of namespace and name. The namespace is a parent name of the variable. The namespace of global attributes is an empty string. This code should ensure empty. If none, it would pick up under class object.